core/domain/feedback_services.py

```diff
@@ -368,7 +421,126 @@ def _get_thread_from_model(thread_model):
         thread_model.id, thread_model.exploration_id, thread_model.state_name,
         thread_model.original_author_id, thread_model.status,
         thread_model.subject, thread_model.summary, thread_model.has_suggestion,
-        thread_model.created_on, thread_model.last_updated)
+        thread_model.message_count, thread_model.created_on,
+        thread_model.last_updated)
+
+
+def get_thread_summaries(user_id, full_thread_ids):
+    """Returns a list of summaries corresponding to each of the threads given.
+
+    Args:
+        user_id: str. The id of the user.
+        full_thread_ids: str. The complete ids of the threads for which we have
+            to fetch the summaries.
+
+    Returns:
+        list(dict). A list of dictionaries containing the summaries of the
+            threads given to it. The format of the returned value:
+            [
+                {
+                    'status': Status of the thread,
```

Comment:
Add type info after each key, e.g. `'status': str. The status of the thread.` End each line with a period.